db/warehouse/migrate/20241207185500_create_hmis_workflows.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+class CreateHmisWorkflows < ActiveRecord::Migration[7.0]
+  def change
+    create_table :wfd_templates do |t|
```

Comment:
I am unsure of how we will differentiate between "CE" WorkflowDefinitions vs other kinds of WorkflowDefinitions (if/when we use these general wfd tables to support other types of workflows). Perhaps we need an additional "type" column for the workflow type that we always populate with `ce_referral` for now? @ttoomey thoughts?

Reply:
Opened https://github.com/greenriver/hmis-warehouse/pull/5284